admin/src/scenes/settings/operations/actions/Desistement/Desistement.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { useToggle } from "react-use";
+import { HiOutlineInformationCircle } from "react-icons/hi";
+import { COHORT_TYPE, CohortDto } from "snu-lib";
+import { Button, Tooltip } from "@snu/ds/admin";
+import DesistementModal from "./DesistementModal";
+
+export default function Desistement({ session }: { session: CohortDto }) {
+  const [showModal, toggleModal] = useToggle(false);
+
+  return (
+    <div className="flex items-center justify-between px-4">
+      <div className="flex gap-2">
+        <div className="text-sm leading-5 font-bold">Désistement des volontaires n'ayant pas confirmé leur présence</div>
+        <Tooltip id="affectation-hts-metropole" title="Désistement (Metropole, hors Corse)">
```

Comment:
```suggestion
        <Tooltip id="desistement-metropole" title="Désistement (Metropole, hors Corse)">
```
pour que la tooltip fonctionne